content/kubermatic/main/installation/install-kkp-CE/add-seed-cluster-CE/_index.en.md

```diff
@@ -1,8 +1,8 @@
 +++
 title = "Add Seed Cluster to Kubermatic Kubernetes Platform (KKP) CE"
+description = "Complete installation procedure of KKP into a pre-existing Kubernetes cluster using KKP’s installer"
```

Comment:
This is not correct, this page only describes sed installation (as per the previous description that got removed).